.gitignore

```diff
@@ -38,3 +38,7 @@ notebooks/*.tgz
 terraform/.terraform/*
 .env
 .hypothesis/
+devtools/datasette/fly/Dockerfile
+devtools/datasette/fly/inspect-data.json
+devtools/datasette/fly/metadata.yml
+devtools/datasette/fly/all_dbs.tar.zst
```

Comment:
Why are these ignored?

Reply:
These are generated by `publish.py` and should not be checked in. I'll put in a comment.

Reply:
Ah of course 👍